helix-core/src/main/java/org/apache/helix/task/TaskUtil.java

```diff
@@ -220,6 +221,85 @@ protected static void setWorkflowContext(HelixManager manager, String workflowRe
   }
 
   /**
+   * Intialize the user content store znode setup
+   * @param manager          a connection to Helix
+   * @param workflowResource the name of workflow
+   * @param jobResource      the name of job
+   * @param record           the initial data
+   */
+  protected static void setUserContent(HelixManager manager, String workflowResource,
```

Comment:
job will be set at same time. It is the initialization for both workflow and job